src/models/payment.js

```diff
@@ -24,6 +24,23 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.STRING,
         allowNull: true,
       },
+      status: {
+        type: DataTypes.VIRTUAL,
+        get() {
+          const currentDate = new Date();
+          const paymentDate = this.getDataValue('date');
+          let dateLimit = new Date(paymentDate);
+
+          dateLimit.setMonth(dateLimit.getMonth() + 1);
+
+          if (paymentDate.getDate() !== dateLimit.getDate()) {
+            const dateCopy = new Date(paymentDate);
+            dateLimit = new Date(dateCopy.getFullYear(), dateCopy.getMonth() + 2, 1);
```

Comment:
Эта часть всегда возвращает первое число, наверно минус потерялся?
тогда будет последний день след месяца
```suggestion
            dateLimit = new Date(dateCopy.getFullYear(), dateCopy.getMonth() + 2, -1);
```

Reply:
Нет, там должен быть +1, чтобы возвращался последний день след месяца.
-1 не возвращает последний день.